src/CorrelationId/CorrelationIdMiddleware.cs

```diff
@@ -55,7 +59,13 @@ public async Task Invoke(HttpContext context, ICorrelationContextFactory correla
                 });
             }
 
-            await _next(context);
+            using (_logger.BeginScope(new Dictionary<string, object>
+            {
+                {nameof(CorrelationContext.CorrelationId), correlationId}
```

Comment:
This is incorrect, you should use  _options.Header instead

Reply:
Unless you want to add in another variable variable and track it through? 
Having the flexibility of being able to change the logging name would be advantageous.

Reply:
Agree, being able to configure this makes sense.